File: src/test/resources/output/java/okhttp/multipart-data.java
OkHttpClient client = new OkHttpClient();

RequestBody body = new MultipartBody.Builder()
        .setType(MultipartBody.FORM)
        .addFormDataPart("foo", "hello.txt",
                RequestBody.create(MediaType.parse("text/plain"), new File("hello.txt")))
        .build()
Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .post(body)
  .addHeader("content-type", "multipart/form-data")
  .build();

Response response = client.newCall(request).execute();

File: src/test/resources/output/node/unirest/multipart-data.js
var fs = require("fs");
var unirest = require("unirest");

var req = unirest("POST", "http://mockbin.com/har");

req.headers({
  "content-type": "multipart/form-data"
});

req.multipart([
  {
    "content-type": "text/plain",
    "body": fs.createReadStream("hello.txt")
  }
]);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
